build/asset-manifest.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
I recommend adding the route build/ on your `.gitignore`, it is not a good practice to keep track of build destination directories (they will change very often)